overlay.nix

```diff
@@ -0,0 +1,5 @@
+final: prev:
+{
+  agenix = prev.callPackage ./pkgs/agenix.nix { };
+  rage = pkgs.callPackage ./pkgs/rage.nix  {};
```

Comment:
pkgs is not defined here, and I don't think we need to include rage in the overlay.
```suggestion
```